modules/notifications/js/app/notifications.js

```diff
@@ -77,5 +77,7 @@ jQuery( document ).on( 'preload.o2', function( bootstrap ) {
 } );
 
 jQuery( document ).on( 'ready.o2', function() {
-	o2.Notifications.open();
+	if ( 'undefined' !== typeof o2.Notifications ) {
+		o2.Notifications.open();
+	}
```

Comment:
I was going to use optional chaining for this, I think [support](https://caniuse.com/?search=optional%20chaining) is good enough
```suggestion
	o2.Notifications?.open();
```

Reply:
Yeah I'm good with optional chaining, I don't think I'd actually used it when I put this PR together.. :)